### ISSUE TITLE
Missing OM translations for all supported languages

### ISSUE THREAD

**Issue by biodranik** [organicmaps/organicmaps#8854]
Here are some missing OM translations in [data/strings/strings.txt](https://github.com/organicmaps/organicmaps/blob/master/data/strings/strings.txt)
[missing_strings.zip](https://github.com/user-attachments/files/16496402/missing_strings.zip)

and in [data/strings/types_strings.txt](https://github.com/organicmaps/organicmaps/blob/master/data/strings/types_strings.txt)
[missing_types_strings.zip](https://github.com/user-attachments/files/16496407/missing_types_strings.zip)

Our user Felix from Telegram is helping with Chinese translations. Any help with other translations is appreciated. 

CC @organicmaps/translations

**Comment by Arthur-GYT** [comment]
I'm looking for French translations ^^

**Comment by NeatNit** [comment]
Is there a way to say "deliberately delegating to English"? e.g. [openstreetmap]:
```
  [openstreetmap]
    comment = Text in menu
    tags = android
    en = OpenStreetMap
    ar = خريطة الشارع المفتوحة
    zh-Hant = 開放街圖
```
those are all the languages defined because everyone else just wants it to say OpenStreetMap and it feels dumb to just duplicate.

This string is perhaps unlikely to ever change, but other cases might, and in those cases isn't it best to leave out the duplicates so that any English change is automatically applied too?

Also, while we're on the topic of general translations: do you think the name Organic Maps should be translated? Based on [about_menu_title] it isn't translated in most languages (unless they happen to have the exact same spelling), with `bg` being the only exception. This is particularly of note for RTL languages like Hebrew where I have to take special care to avoid putting LTR text at the start of a string.

**Comment by biodranik** [comment]
@NeatNit Duplicates are automatically cleaned up by the script (android) or generated on iOS where necessary.

Use something like `comment = Text in menu, should be translated only if OpenStreetMap project is widely recognized in your language under a different name`

Translating Organic Maps name is unnecessary in most languages, where inserting English characters/names works for readers.

**Comment by muralito** [comment]
I have a similar question. One report says es-MX has 518 missing strings. ¿should the missing strings be duplicated from "es"? ¿Or it can be "delegated to es" assuming "es" is for generic spanish, and Spain localized spanish is coded "es-ES")?

**Comment by NeatNit** [comment]
@biodranik Okay, as long as we're clear that the goal isn't to get down to 0 but rather to make us aware of the state of our translations.

**Comment by biodranik** [comment]
The current approach to highlight missing strings doesn't work well for language variants. On the other hand, duplicating everything is also an overkill.

**Comment by matheusgomesms** [comment]
I know some devs are reluctant (#4515) but as the sole translator of PT/PT-BR, I find it extremely difficult to keep-up with translations without a tool such as -_insert here any translation tool_-. Not sure if it productive of my free (and volunteered) time to check 600 strings, in .txt files, to see which one is duplicate or missing. Also, not only these strings are missing, but there are lots of other strings that were probably AI generated and there is no easy way to know them.

Already discussed _ad nauseam_, but all of this could be easily solved with any tool (2-3 clicks max). While OM don't solve this (which is the norm in all apps, by the way), big no from me.

**Comment by HuggeK** [comment]
> I know some devs are reluctant (#4515) but as the sole translator of PT/PT-BR, I find it extremely difficult to keep-up with translations without a tool such as -_insert here any translation tool_-. Not sure if it productive of my free (and volunteered) time to check 600 strings, in .txt files, to see which one is duplicate or missing. Also, not only these strings are missing, but there are lots of other strings that were probably AI generated and there is no easy way to know them.
> 
> 
> 
> Already discussed _ad nauseam_, but all of this could be easily solved with any tool (2-3 clicks max). While OM don't solve this (which is the norm in all apps, by the way), big no from me.

I agree, there is this txt in [TRANSLATIONS.md](https://github.com/organicmaps/organicmaps/blob/master/docs/TRANSLATIONS.md):

>There are preliminary plans to refactor translations workflow and migrate to Weblate.

Is there something we could help to move to webplate or a similar tool? Is the hostning ok on their side? Do we need to refractor code before we use it?

**Comment by biodranik** [comment]
The only blocker in moving to Weblate or any other tool is missing support of the current OM workflow with txt files (twine-based). E.g. a twine-compatible plugin to Weblate would unblock it immediately. It will allow supporting of two-way workflows (from weblate to OM, and from OM to weblate) seamlessly while keeping the current review process (which is CRITICALLY important for good quality translations!) and tooling to detect wrong/bad strings and keep them clean.

Our experience with Weblate so far is negative, both in terms of workflow convenience for core contributors and translation quality. It was discussed many times in referenced issues and in other places.

The best quality translations so far were provided by contributors here, on GitHub, or by users sending us updated text files.

Please write all your suggestions in other relevant issues and leave this one focused on missing strings.
